homeassistant/components/comelit/strings.json

```diff
@@ -3,19 +3,25 @@
     "flow_title": "{host}",
     "step": {
       "reauth_confirm": {
-        "description": "Please enter the correct PIN for {host}",
         "data": {
           "pin": "[%key:common::config_flow::data::pin%]"
+        },
+        "data_description": {
+          "pin": "The PIN for your Comelit device {host}"
```

Comment:
1. I am not sure if placeholders work in the description
2. I think we can use the entry title instead and just mention it in the description? "You have to reauthenticate to get {title} to work again" (not perfect, but as an example)

Reply:
Copied the description for below